src/rl_framework/util/saving_and_loading/connector/clearml_connector.py

```diff
@@ -134,17 +135,13 @@ def upload(
             # TODO: Save README.md
 
     def download(self, *args, **kwargs) -> Path:
-        task_id = self.download_config.task_id
+        # noinspection PyUnresolvedReferences
+        model_id = self.download_config.model_id
         file_name = self.download_config.file_name
 
-        assert task_id, file_name
+        assert model_id, file_name
```

Comment:
This is my mistake, but you can fix it:
This requires two asserts.
Otherwise `file name` is the error message.